pkg/edit/vars.go

```diff
@@ -57,6 +76,32 @@ func addVars(fm *eval.Frame, m vals.Map) error {
 	return nil
 }
 
+func delVars(fm *eval.Frame, m vals.List) error {
+	for it := m.Iterator(); it.HasElem(); it.Next() {
+		n := it.Elem()
+		name, ok := n.(string)
+		if !ok {
+			return errs.BadValue{
+				What:  "name of argument to edit:del-vars",
```

Comment:
`name of argument to edit:del-vars` doesn't seem to be an accurate description of what the element is; `element of the argument to edit:del-vars` is better.

I think it's better to first convert the argument into a `[]string` and then delete all the arguments. Deleting them in a loop is prone to race conditions.